libimage/layer_tree.go

```diff
@@ -31,10 +32,19 @@ func (t *layerTree) node(layerID string) *layerNode {
 	return node
 }
 
+// ErrorIsUnknown returns true if the specified error indicates that an
+// image is unknown or has been partially removed (e.g., a missing layer).
+func ErrorIsUnknown(err error) bool {
```

Comment:
Cringe. To a naïve reader, this function name reads as "Unknown error type", not unknown image/layer/something. Even reading the comment and code, my brain still cannot parse `ErrorIsUnknown` the way you intend it. Could you please reconsider this name before it's cast in stone?

Reply:
Great comment, thanks!  I'll rename.

Reply:
Renamed to `ErrorIsImageUnknown` which should be clearer to new readers.